velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -722,6 +727,20 @@ RowTypePtr VectorFuzzer::randRowType(int maxDepth) {
 }
 
 VectorPtr VectorFuzzer::wrapInLazyVector(VectorPtr baseVector) {
+  if (hasNestedDictionaryLayers(baseVector)) {
+    auto indices = baseVector->wrapInfo();
+    auto values = baseVector->valueVector();
+    auto nulls = baseVector->nulls();
+
+    auto copiedIndices = AlignedBuffer::copy(baseVector->pool(), indices);
```

Comment:
Why do we need to copy?

Reply:
I was being a bit too cautious in trying to separate this copy completely from the input vector in case the original somehow has its buffer modified when it goes through its expression execution cycle. Ideally that should not happen if i use the same buffer as it would not be unique.
I can switch it to using the same buffer